test/e2e/serial/tests/non_regression_fundamentals.go

```diff
@@ -19,16 +19,20 @@ package tests
 import (
 	"context"
 	"fmt"
+	e2eclient "github.com/openshift-kni/numaresources-operator/test/utils/clients"
```

Comment:
minor: please add import to the proper group (sdlib imports, k8s imports, 3rd party imports, local imports...)